### PR TITLE
[pipelineX](sink) support jdbc table sink

### PR DESCRIPTION
**Pull request by zhangstar333**
## Proposed changes
support jdbc table sink in pipelineX

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zhangstar333** [comment]
run buildall

File: be/src/pipeline/exec/jdbc_table_sink_operator.cpp [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+namespace doris {
+class DataSink;
+} // namespace doris
+
+namespace doris::pipeline {
+
+JdbcTableSinkOperatorX::JdbcTableSinkOperatorX(const RowDescriptor& row_desc,
+                                               const std::vector<TExpr>& t_output_expr)
+        : DataSinkOperatorX(0), _row_desc(row_desc), _t_output_expr(t_output_expr) {
+    _name = "JdbcTableSink";
```
Comment (Gabriel39) [inline be/src/pipeline/exec/jdbc_table_sink_operator.cpp:36]: Do not set `name` here. We set name in base class

**Comment by zhangstar333** [comment]
run buildall

**Review by Gabriel39 — APPROVED** [review]
LGTM

**Review by BiteTheDDDDt — APPROVED** [review]

**Pull request merged**